setup.py

```diff
@@ -100,7 +100,7 @@ def get_git_sha():
         'retry>=0.9.2',
         'selenium>=3.141.0',
         'simplejson>=3.15.0',
-        'sqlalchemy',
+        'sqlalchemy>=1.2.18, <2.0.0',
```

Comment:
`sqlalchemy==1.3.0` just came out and is having some teething problems. I would suggest restricting to `<1.3.0` until those are sorted out, otherwise `pip` will pull in the 1.3 version.